packages/devtools/test/support/cli_test_driver.dart

```diff
@@ -103,6 +103,9 @@ class CliAppFixture extends AppFixture {
 
     final VM vm = await serviceConnection.getVM();
 
+    await Future.wait(
```

Comment:
We might be hitting a race her:

```
01:19 +4 -1: test/integration_tests/integration_test.dart: integration logging displays log data [E]
  Isolate must be runnable (105) from resume():
  Isolate must be runnable before this request is made.
  dart:async                                           _Completer.completeError
  package:vm_service_lib/vm_service_lib.dart 861:17    VmService._processResponse
  package:vm_service_lib/vm_service_lib.dart 848:7     VmService._processMessageStr
  package:vm_service_lib/vm_service_lib.dart 800:7     VmService._processMessage
```

The isolate might exist, but not yet be runnable (it goes through a small life cycle at startup).

We could continue re-requesting isolates until we got one that was in the paused-at-start state?

Reply:
Heh, how ironic :-) Neither the original race nor this occurred locally so I had to debug-via-Travis (which is somewhat tedious!).

Yeah, I'll put a little loop in for now. The nicer fix would probably be to subscribe to the event stream, then check, and if it's not already ready then wait for the event on the stream, but it's probably overly complex for this, so I'll just use `waitFor` or similar.